organization/_posts/2017-09-21-startup.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Weekly Meeting #109, September 21, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #109, September 21, 2017
+
+#### *Present*: Ian Bird, Michel Jouvin, Dario Menasce, Pere Mato, Liz Sexton-Kennedy, John Harvey, Benedikt Hegner, Marko Petric, Graeme Stewart, Andre Sailer, Daniel Elvira, David Lange, Mike Sokoloff, Pete Elmer, David Lange, Marco Brako, Thomas Kuhr
```

Comment:
Marco Brako was a person connected remotely?

Reply:
Yes - I grabbed his name from Vidyo. He's in Belle II.